src/peft/tuners/tuners_utils.py

```diff
@@ -168,7 +168,7 @@ def __init__(
         if not hasattr(self, "peft_config"):
             self.peft_config = {adapter_name: peft_config} if isinstance(peft_config, PeftConfig) else peft_config
         else:
-            logger.info(
+            logger.warning(
```

Comment:
@BenjaminBossan  Based on youur previous msg, peft should only do warning, and it looks like a warning message to me, not level == info.

Reply:
done